code/processes/rdb.q

```diff
@@ -166,6 +172,9 @@ getpartition:{[] rdbpartition}
 notpconnected:{[]
 	0 = count select from .sub.SUBSCRIPTIONS where proctype in .rdb.tickerplanttypes, active}
 
+/-resets timeout to 0 before EOD writedown
+timeoutreset:{.rdb.timeout:system"T";system"T 0"};
+restoretimeout:{system["T ", string timeout]};
```

Comment:
For clarity, i would call this .rdb.timeout rather than just timeout (given that in the line above you have referenced .rdb.timeout)